src/ga/ga_simple.rs

```diff
@@ -101,7 +101,9 @@ impl<T: GAIndividual + Clone> GeneticAlgorithm<T> for SimpleGeneticAlgorithm <T>
         let best_old_individual = self.population.best().clone();
 
         // Evaluate the new population
-        self.population.swap(new_individuals);
+        // TODO: Archive the old population
```

Comment:
We will be archiving the statistics object of every population, see [here](https://github.com/mrjackinc/rust-monster/blob/272237ffa3c84ba5460cfe527dc681a275154cab/src/ga/ga_statistics.rs#L90). If storing only the **best** solution object suffices, we can add that to the [GAPopulationStats](https://github.com/mrjackinc/rust-monster/blob/272237ffa3c84ba5460cfe527dc681a275154cab/src/ga/ga_population.rs#L456) object. The history of best solutions would thus be stored.